Update Versioning.md to include information about the --follow-tags flag

nrwinner opened the pull request.

Commit:
Update Versioning.md

DonSeannelly reviewed (commented):
Could we perhaps amend this to say if your git installation is not already configured to follow tags by default, then add the flag?

Commit:
Update Versioning.md

Update Versioning to include information about configuring a git installation to always push existing local tags to remote.

Commit:
Update Versioning.md

Fix spacing issues to give commands their own line

DonSeannelly reviewed (approved):
Thanks so much for taking the time to add this! 💯
